src/components/ProgressIndicator/index.jsx

```diff
@@ -23,11 +23,6 @@ const ProgressIndicator = ({
   const [resizeCounter, setResizeCounter] = useState(0);
   const buttonRef = useRef(null);
 
-  const modifiers = [];
-  if (expanded) {
-    modifiers.push('expanded');
-  }
```

Comment:
Couldn't find any CSS anymore that still did something with it, I reckon we missed this when we rewrote the progress indicator.